scripts/utils/generateDocs.js

```diff
@@ -140,13 +143,29 @@ function generateApiDoc (spec) {
   return doc
 }
 
+const LINK_OVERRIDES = {
+  "license.delete": "{ref}/delete-license.html",
+  "license.get": "{ref}/get-license.html",
+  "license.get_basic_status": "{ref}/get-basic-status.html",
+  "license.get_trial_status": "{ref}/get-trial-status.html",
+  "license.post": "{ref}/update-license.html",
+  "license.post_start_basic": "{ref}/start-basic.html",
+  "license.post_start_trial": "{ref}/start-trial.html",
+  "migration.deprecations": "{ref}/migration-api-deprecation.html",
+  "monitoring.bulk": "{ref}/es-monitoring.html",
+}
 // Fixes bad urls in the JSON spec
-function fixLink (str) {
+function fixLink (name, str) {
+  var override = LINK_OVERRIDES[name]
+  if (override) return override
   if (!str) return ''
-  if (str.includes('/5.x/')) {
```

Comment:
Elasticsearch has all kinds of wrong branches linked and this new replacement forces them all to the "current" branch.